server/middlewares/statsRedis.js

```diff
@@ -56,20 +55,8 @@ const trackTxSubmissions = mung.json((body, req, res) => {
     incrCountersBy(`${txSubmissionType}:${txSubmissionSuccess}`, 1)
 
     if (txSubmissionSuccess === 'successful') {
-      const {txBody} = req.body
-      let txOutAmount = 0
-      let txTotalAmount = 0
-
-      try {
-        txOutAmount = parseTxBodyOutAmount(txBody)
-        txTotalAmount = parseTxBodyTotalAmount(txBody)
-      } catch (e) {
-        // eslint-disable-next-line no-console
-        console.error(e)
-      }
-
-      incrCountersBy(`${txSubmissionType}:sentOut`, txOutAmount)
-      incrCountersBy(`${txSubmissionType}:sentTotal`, txTotalAmount)
+      incrCountersBy(`${txSubmissionType}:sentOut`, 0)
```

Comment:
I would just omit the counters if it's not working anyway

Reply:
same for statsGoogleAnalytics where I see that the value is commented out right now